src/pyhf/utils.py

```diff
@@ -98,13 +98,10 @@ def qmu(mu, data, pdf, init_pars, par_bounds):
     Args:
         mu (Number or Tensor): The signal strength parameter
         data (Tensor): The data to be considered
-        pdf (|pyhf.pdf.Model|_): The HistFactory statistical model used in the likelihood ratio calculation
+        pdf (:func:`pyhf.pdf.Model`): The HistFactory statistical model used in the likelihood ratio calculation
```

Comment:
@kratsg Do you know if it is possible for us to use this nicer API but still keep the particular styling where we get `pyhf.pdf.Model` (like we have now below) and not `pyhf.pdf.Model()`?

![nice_look](https://user-images.githubusercontent.com/5142394/70081154-a04b7480-15cd-11ea-879c-d3389c96255a.png)

Reply:
Just don't use the `:func:` for parameter types. You only use it when you need to explicitly force linking.

Reply:
![Screenshot 2019-12-03 12 15 48](https://user-images.githubusercontent.com/761483/70086204-bb66b600-15c6-11ea-810c-6c45e4e6ae69.png)

is done with

```
        Raises:
          ~pyhf.exceptions.InvalidWorkspaceOperation: The workspaces have common channel names, incompatible measurements, or incompatible schema versions.

        Args:
            left (~pyhf.workspace.Workspace): A workspace
            right (~pyhf.workspace.Workspace): Another workspace

        Returns:
            ~pyhf.workspace.Workspace: A new combined workspace object
```

Reply:
I used `:obj:` instead to maintain the entire namespace of `pyhf.pdf.Model`. Would you prefer it just be `Model` with `~pyhf.pdf.Model`?

Reply:
Yes, I prefer the `~pyhf.pdf.Model`. In documentation types like this, you only need the short name of the class rather than the full path. The full path will be on the class' doc page anyway.